### DISCUSSION TITLE
Android 11 - Rendering Issue

### DISCUSSION THREAD

**Discussion by sudhans** [FreeRDP/FreeRDP#7909] [Q&A]
Hi, My android client that uses freerdp 2.6.1 works on Android 6 and has rendering issues on Android 11 when connected to a Windows 10 desktop. Has anyone faced this issue? Any suggestions, workarounds, solutions are welcome. I have attached a screenshot and highlighted the areas with a GREEN rectangle. 
![image](https://user-images.githubusercontent.com/1388504/168820260-eb88b39b-4bbc-46c3-82f3-748a73407bf4.png)

**Comment by akallabeth** [comment]
How do you compile?
Android can have the following `AVC`  (`=H264`) support options:
1. Android Mediacodec (dependent on device, might be buggy on one, working on the next, or just some resolutions)
2. `OpenH264`: Update `FreeRDP`, we´ve fixed a bug preventing versions `>= 1.8` from properly displaying the contents
3. `FFMPEG` (which in turn uses `x264` or `openh264`)
